### ISSUE TITLE
Installer improvement suggestion

### ISSUE THREAD

**Issue by AlexIbanez16** [BabylonJS/Exporters#899]
Hello,

I'm writing a suggestion concerning the installer.

Using the exporter on a daily basis, it's important for me to keep the exporter up to date and functioning.
One day, however, I had a problem with an update of the exporter that wouldn't let me export. I tried to go back to a previous version, but it was tedious manually and I lost a lot of time.

What I suggest is that instead of having the installer automatically installing the latest version, maybe make an installer that installs only the version from when it was posted on GitHub. This way, if I want to go back to an earlier version, I just have to download the old installer on GitHub.

An alternative would be to select which version of the exporter you want to install directly on the installer. That could be even better since we wouldn't have to download another installer, and it would automatically check the versions available.

I hope you find this suggestion helpful.

**Comment by Drigax** [comment]
Definitely. I think it would be a good idea to allow users to install previous versions, especially to make recovering from issues easier. This isn't a high priority ask for us, however but I'd be happy to check in any additions made to the installer.
